### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by erolaliyev**

### CODE REVIEW

**Review by isaac-tuagba — APPROVED** [review]
You should add wrap the button in the course card component with the Link tag to enable routing to the course page

File: src/components/Courses/components/CourseCard/CourseCard.jsx [hunk 1]
```diff
@@ -35,4 +37,12 @@ const CourseCard = ({
 	);
 };
```
[the remark this answers is not in the document]
Comment (erolaliyev) [reply src/components/Courses/components/CourseCard/CourseCard.jsx:38]: Do you mean the Show course button in each course card? It wasn't mentioned in the task description. That's why I thought maybe in the next tasks they will ask to implement this using GET request.

[the remark this answers is not in the document]
Comment (isaac-tuagba) [reply src/components/Courses/components/CourseCard/CourseCard.jsx:38]: Its actually mentioned in the task 
CourseCard Component
[1 point] - Add functionality for button Show course on CourseCard component.
When user clicks Show course button he should be navigated to the page with course information.

[the remark this answers is not in the document]
Comment (erolaliyev) [reply src/components/Courses/components/CourseCard/CourseCard.jsx:38]: Oh sorry about that, my bad. I only check task descriptions pdf for tasks. I will check the grading pdf too from now on. Thanks for pointing it out. 👍

**Pull request merged**